### ISSUE TITLE
"storybook test --coverage" is not printing coverage to console

### ISSUE THREAD

**Issue by jnahumphreys** [storybookjs/test-runner#454]
### Describe the bug

On running `npm test-storybook --coverage` the coverage report is failing to print to the console (tests run fine btw)

### Steps to reproduce the behavior

1. Create a component built using TypeScript
2. Write stories for said component
3. start storybook `npm run storybook`
4. run tests with coverage flag for storybook `npm run storybook --coverage`

### Expected behavior

My tests should run and print the coverage report to my console once all tests are comple

### Screenshots and/or logs

```
Ran all test suites.
Coverage file (134643 bytes) written to .nyc_output/coverage.json
nyc report

run coverage report for .nyc_output

Options:
  --cwd                   working directory used when resolving paths
                           [string] [default: "/Users/james/Development/Personal
                                           Projects/binaryjim-github-user-site"]
  --nycrc-path            specify an explicit path to find nyc configuration
  --temp-dir, -t, -t      directory to output raw coverage information to
                                             [string] [default: "./.nyc_output"]
  --exclude, -x, -x       a list of specific files and directories that should
                          be excluded from coverage, glob patterns are supported
                                                              [string] [default:
  ["coverage/**","packages/*/test{,s}/**","**/*.d.ts","test{,s}/**","test{,-*}.{
  js,cjs,mjs,ts,tsx,jsx}","**/*{.,-}test.{js,cjs,mjs,ts,tsx,jsx}","**/__tests__/
  **","**/{ava,babel,nyc}.config.{js,cjs,mjs}","**/jest.config.{js,cjs,mjs,ts}",
         "**/{karma,rollup,webpack}.config.js","**/.{eslint,mocha}rc.{js,cjs}"]]
  --exclude-node-modules  whether or not to exclude all node_module folders
                          (i.e. **/node_modules/**) by default
                                                       [boolean] [default: true]
  --include, -n, -n       a list of specific files that should be covered, glob
                          patterns are supported          [string] [default: []]
  --extension, -e, -e     a list of extensions that nyc should handle in
                          addition to .js
                   [string] [default: [".js",".cjs",".mjs",".ts",".tsx",".jsx"]]
  --exclude-after-remap   should exclude logic be performed after the source-map
                          remaps filenames?            [boolean] [default: true]
  --branches              what % of branches must be covered?
                                                           [number] [default: 0]
  --functions             what % of functions must be covered?
                                                           [number] [default: 0]
  --lines                 what % of lines must be covered?[number] [default: 90]
  --statements            what % of statements must be covered?
                                                           [number] [default: 0]
  --per-file              check thresholds per file   [boolean] [default: false]
  --check-coverage        check whether coverage is within thresholds provided
                                                      [boolean] [default: false]
  --reporter, -r, -r      coverage reporter(s) to use
                                                    [string] [default: ["text"]]
  --report-dir            directory to output coverage reports in
                                                  [string] [default: "coverage"]
  --show-process-tree     display the tree of spawned processes
                                                      [boolean] [default: false]
  --skip-empty            don't show empty files (no lines of code) in report
                                                      [boolean] [default: false]
  --skip-full             don't show files with 100% statement, branch, and
                          function coverage           [boolean] [default: false]
  -h, --help              Show help                                    [boolean]
  --version               Show version number                          [boolean]

Examples:
  nyc report --reporter=lcov  output an HTML lcov report to ./coverage

Too many non-option arguments: got 2, maximum of 0
/Users/james/Development/Personal Projects/binaryjim-github-user-site/node_modules/@storybook/test-runner/bin/test-storybook.js:26
  throw err;
  ^

Error: Command failed: npx nyc report --reporter=text -t /Users/james/Development/Personal Projects/binaryjim-github-user-site/coverage/storybook --report-dir /Users/james/Development/Personal Projects/binaryjim-github-user-site/coverage/storybook
    at checkExecSyncError (node:child_process:841:11)
    at execSync (node:child_process:912:15)
    at reportCoverage (/Users/james/Development/Personal Projects/binaryjim-github-user-site/node_modules/@storybook/test-runner/bin/test-storybook.js:73:3)
    at process.onProcessEnd (/Users/james/Development/Personal Projects/binaryjim-github-user-site/node_modules/@storybook/test-runner/bin/test-storybook.js:80:3)
    at process.emit (node:events:525:35)
    at process.el [as emit] (/Users/james/Development/Personal Projects/binaryjim-github-user-site/node_modules/playwright-core/lib/utilsBundleImpl.js:14:10212) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 75613,
  stdout: null,
  stderr: null
}
```

### Environment

 - OS: MacOS 12.6
 - Node.js version: v16.17.0
 - NPM version: 8.19.2
 - Browser (if applicable): N/A
 - Browser version (if applicable): N/A
 - Device (if applicable): N/A

### Additional context

My Typescript has been instrumented successfully, after running `npm run storybook --coverage` I can generate an lcov report by running `npx nyc report --reporter=lcov -t coverage/storybook --report-dir coverage/storybook`.

**Comment by yannbf** [comment]
Hey everyone, the issue is now fixed in 0.18.0 #456. Thanks!

**Comment by yannbf** [comment]
@edwardscull I appreciate the detail you put in your comment!

> - I wold like to generate HTML reports and not just a single JSON file
> - I would like to see the terminal printout showing the results

These are default in 0.18.0

> I would like to see all my components in the coverage results. Currently I am only seeing 2 of my components

You might need to use the istanbul options in the addon coverage to include different directories: https://github.com/storybookjs/addon-coverage#configuring-the-addon
